BaragonServiceIntegrationTests/src/test/java/com/hubspot/baragon/BaragonServiceTestIT.java

```diff
@@ -0,0 +1,478 @@
+package com.hubspot.baragon;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import com.hubspot.baragon.client.BaragonServiceClient;
+import com.hubspot.baragon.models.BaragonAgentMetadata;
+import com.hubspot.baragon.models.BaragonRequest;
+import com.hubspot.baragon.models.BaragonRequestState;
+import com.hubspot.baragon.models.BaragonResponse;
+import com.hubspot.baragon.models.BaragonService;
+import com.hubspot.baragon.models.BaragonServiceState;
+import com.hubspot.baragon.models.BaragonServiceStatus;
+import com.hubspot.baragon.models.UpstreamInfo;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.Stopwatch;
+import org.junit.rules.TestRule;
+import org.junit.rules.TestWatcher;
+import org.junit.runner.Description;
+
+import com.google.common.base.Optional;
+
+import static org.junit.Assert.*;
+
+public class BaragonServiceTestIT {
+  
+  public static final String LOAD_BALANCER_GROUP = "docker-baragon";
+  public static final String UPSTREAM = "example.com:80";
+  public static final String SERVICE_BASE_PATH = "/testservice";
+	
+  private BaragonServiceClient baragonServiceClient;
+
+  // Helpers --------------------------------------------
+  
+  private Injector getInjector() {
+    return Guice.createInjector(new DockerTestModule());
+  }
+  
+  private String setupTestService() {
+    String requestId = "Service" + new Date().getTime();
+    setupTestService(requestId, requestId, SERVICE_BASE_PATH, Optional.<String>absent(), Optional.<Map<String, Object>>absent(), Optional.<String>absent());
+    
+    return requestId;
+  }
+  
+  private void setupTestService(String requestId, String serviceId, String basePath, Optional<String> replaceServiceId, Optional<Map<String, Object>> options, Optional<String> template) {
+    System.out.println("Requesting " + requestId + "...");
```

Comment:
Let's use the same logger we use for the rest of Baragon rather than System.out, for example https://github.com/HubSpot/Baragon/blob/master/BaragonService/src/main/java/com/hubspot/baragon/service/managed/BaragonManaged.java#L22